Blackjack.xcodeproj/project.pbxproj

```diff
@@ -158,12 +176,18 @@
 			isa = PBXSourcesBuildPhase;
 			buildActionMask = 2147483647;
 			files = (
-				04837BFD2385D8E9001D8F60 /* Royals.swift in Sources */,
+				041E5C3323870B9100E78B7A /* Hand.swift in Sources */,
 				041C6A4A23467CD6000A5CBC /* ViewController.swift in Sources */,
+				0498C3F023FC58A80009D480 /* PlayerStatus.swift in Sources */,
+				0498C3F223FC59720009D480 /* Game.swift in Sources */,
+				04961EE82388AD830070EE1D /* LivePlayer.swift in Sources */,
 				041C6A4623467CD6000A5CBC /* AppDelegate.swift in Sources */,
+				0461332E2387532300280855 /* CardTitles.swift in Sources */,
 				040E2B5723830E17000BCEDF /* Deck.swift in Sources */,
 				041C6A4823467CD6000A5CBC /* SceneDelegate.swift in Sources */,
 				04837BFF2385DB92001D8F60 /* Suits.swift in Sources */,
+				04961EE62388AB140070EE1D /* Player.swift in Sources */,
+				041B448D2396DB4600B5879B /* Dealer.swift in Sources */,
 				0414AAE5234CF9BE00EE0FCB /* Card.swift in Sources */,
 			);
 			runOnlyForDeploymentPostprocessing = 0;
```

Comment:
not sure what this, looks like database or compiler migration, so looks okay to me.